tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -1096,7 +1096,7 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTestAirCooled)
                     ObjexxFCL::Optional_int_const(),
                     ObjexxFCL::Optional<const Real64>());
 
-    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 3.87); // 13.20 IPLV
+    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 3.89); // 13.20 IPLV
```

Comment:
IPLV values change due to correction to the condenser entering air temperature per the AHRI STD.